onnx/version_converter/convert.h

```diff
@@ -523,6 +524,7 @@ class DefaultVersionConverter : public BaseVersionConverter {
     registerAdapter(make_unique<CompatibleAdapter>("OptionalHasElement", OpSetID(17), OpSetID(18)));
     registerAdapter(make_unique<CompatibleAdapter>("ScatterND", OpSetID(17), OpSetID(18)));
     registerAdapter(make_unique<CompatibleAdapter>("ScatterElements", OpSetID(17), OpSetID(18)));
+    registerAdapter(make_unique<MaxPool_17_18>());
```

Comment:
You can used the predefined parametrized adapter `RemoveAttribute` ... no need to define a new adapter class.
```cpp
   registerAdapter("MaxPool", 17, 18, RemoveAttribute(kstorage_order));
```

Reply:
Good point, done.